src/zenml/zen_server/routers/steps_endpoints.py

```diff
@@ -78,7 +78,7 @@ def create_run_step(
     Returns:
         The created run step.
     """
-    return zen_store().create_run_step(step=step)
+    return zen_store().create_run_step(step_run=step)
```

Comment:
Yes, this is still inconsistent in several places. In 247 places we call it `step_run`, in 83 we call it `run_step`. Seems like a major refactoring with little impact, so I would skip it for now. The main reason I renamed it here is because `step_id` etc. might refer to another table in the future.